platform/viewer/src/connectedComponents/ConnectedCineDialog.js

```diff
@@ -16,7 +17,8 @@ const { setViewportSpecificData } = OHIF.redux.actions;
 const mapStateToProps = state => {
   // Get activeViewport's `cine` and `stack`
   const { viewportSpecificData, activeViewportIndex } = state.viewports;
-  const { cine, dom } = viewportSpecificData[activeViewportIndex] || {};
+  const { cine } = viewportSpecificData[activeViewportIndex] || {};
+  const dom = commandsManager.runCommand('getActiveViewportEnabledElement');
```

Comment:
Ah. Yes. This right here.
Can we duplicate this in the `vtk` code?
(Maybe not until we inject the commandsManager?)

Reply:
Question... We should wrap commandsModule like init and inject the commandsManager?

Reply:
I think so. We can pass it in the same object we pass serviceManager

Reply:
Got it. Done! thx.